### PR TITLE
Update logo

### PR DESCRIPTION
**Pull request by kuzminadya**
tailor-old-new

### CODE REVIEW

File: logo/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+<img src="https://raw.githubusercontent.com/zalando/tailor/logo/tailor-logo.svg" alt="Tailor" width="500" height="200">
+
+Download as [SVG](https://raw.githubusercontent.com/zalando/tailor/logo/tailor-logo.svg).
+
+## Credits
+
+The Tailor logo was designed by [Nadya Kuzmina](http://nadyakuzmina.com/).  
+
+## License
```
Comment (vigneshshanmugam) [inline logo/README.md]: Can you change it to logo license?.. Tailor license itself is MIT.

**Comment by vigneshshanmugam** [comment]
@kuzminadya Thanks for the awesome logo 💃

**Comment by w0rm** [comment]
@vigneshshanmugam FYI https://github.com/reactjs/redux/tree/master/logo was used as a reference

**Comment by vigneshshanmugam** [comment]
@w0rm 👍

**Pull request merged**